Presentation.Web/Models/API/V1/Organizations/InterfaceExposedOnSystemsOutsideTheOrganizationConflictDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Presentation.Web.Models.API.V1.Organizations
+{
+    public class InterfaceExposedOnSystemsOutsideTheOrganizationConflictDTO
```

Comment:
I'm wondering what is the purpose of this class, and what is it used for?

Reply:
@a-naskret It's used on the main conflicts DTO:
![image](https://user-images.githubusercontent.com/13116889/166669391-b342fcc3-9b89-4f78-91a0-049142b825c2.png)